### PR TITLE
test: add additional test for BP_NPM_START_SCRIPT

### PR DESCRIPTION
**Pull request by mhdawson**
- add an additional unit test for BP_NPM_START_SCRIPT

## Summary

- Add an additional test for BP_NPM_START_SCRIPT which was added and documented
  a while back

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [X] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [X] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: build_test.go [hunk 1]
```diff
@@ -253,6 +254,33 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 
 			Expect(startScript).To(matchers.BeAFileWithSubstring("some-prestart-command && some-start-command && some-poststart-command"))
 		})
+
+		context("when the project-path env var is not set", func() {
```
Comment (pacostas) [inline build_test.go:258]: How the project-path is related/affects the `BP_NPM_START_SCRIPT` ? 
Also is already on the same context.

Comment (mhdawson) [inline build_test.go:258]:
````suggestion
		context("when BP_NMP_START_SCRIPT is used", func() {
```

Comment (mhdawson) [reply build_test.go:258]: Good catch, cut and paste error.

Comment (mhdawson) [reply build_test.go]: @pacostas fixed

**Review by pacostas — APPROVED** [review]
LGTM

**Pull request merged**